extensions/azdata/package.json

```diff
@@ -20,17 +20,66 @@
   },
   "main": "./out/extension",
   "contributes": {
-    "configuration":  {
-      "type": "object",
-      "title": "%azdata.config.title%",
-      "properties": {
-        "azdata.logDebugInfo": {
-          "type": "boolean",
-          "default": false,
-          "description": "%azdata.config.debug%"
+    "configuration": [
+      {
+        "type": "object",
+        "title": "%azdata.config.title%",
+        "properties": {
+          "azdata.logDebugInfo": {
+            "type": "boolean",
+            "default": false,
+            "description": "%azdata.config.debug%"
+          },
+          "azdata.acceptEula": {
+            "type": "string",
+            "default": "Ask Later",
+            "enum": [
+              "Don't Ask Again",
+              "Ask Later"
+            ],
+            "description": "%azdata.acceptEula.description%"
+          },
+          "azdata.auto.install": {
+            "type": "string",
+            "default": "Ask Later",
```

Comment:
Let's simplify these to `prompt` and `dontPrompt` and then use the descriptions to explain what they each mean